endpoints/api/tag.py

```diff
@@ -229,6 +229,9 @@ def delete(self, namespace, repository, tag):
 
         registry_model.delete_tag(repo_ref, tag)
 
+        if app.config.get("FEATURE_QUOTA_MANAGEMENT", False):
+            repository.force_cache_repo_size(repo_ref.id)
```

Comment:
repository is a string here. `force_cache_repo_size` on repository(str obj) will fail.